src/app/Http/Requests/UserUpdateCrudRequest.php

```diff
@@ -2,34 +2,21 @@
 
 namespace Backpack\PermissionManager\app\Http\Requests;
 
-use Illuminate\Foundation\Http\FormRequest;
+use Backpack\CRUD\app\Http\Requests\CrudRequest;
 
-class UserUpdateCrudRequest extends FormRequest
+class UserUpdateCrudRequest extends CrudRequest
 {
     /**
-     * Determine if the user is authorized to make this request.
```

Comment:
Take a look at `Backpack\CRUD\app\Http\Requests\CrudRequest` it already has that check, so no reason to duplicate the code.